shopify/base.py

```diff
@@ -17,9 +17,6 @@
 class ShopifyConnection(pyactiveresource.connection.Connection):
     response = None
 
-    def __init__(self, site, user=None, password=None, timeout=None, format=formats.JSONFormat):
```

Comment:
These got flagged as linting errors because they replicate the parent's signature, so they're not changing anything.